ansible/prater.yml

```diff
@@ -46,7 +46,10 @@
       vars:
         # TEMPORARY switch for Prater Altair transition
         beacon_node_service_name: 'beacon-node-{{ beacon_node_network }}-{{ node.branch }}'
-        beacon_node_repo_branch: 'unstable'
+        beacon_node_repo_branch:  |-
+          {{ (node.branch == 'stable') | ternary("unstable-revert-taskpool", "unstable") }}
+        beacon_node_threads:  |-
+          {{ (node.branch == 'testing') | ternary(4, 1) }}
```

Comment:
I assume you are aware this change only affects systemd services on linux hosts.

Reply:
Yes, this is a short-lived experiment and these hosts will be enough for us to obtain the necessary information.

Reply:
Okay.